frontend/src/libs/components/app/app.tsx

```diff
@@ -7,19 +7,45 @@ import {
   useAppSelector,
   useEffect,
   useLocation,
+  useMemo,
+  useNavigate,
 } from '~/libs/hooks/hooks.js';
 import { actions as userActions } from '~/slices/users/users.js';
 
+import { iconNameToSvg } from '../icon/maps/maps.js';
+
 const App: React.FC = () => {
   const { pathname } = useLocation();
   const dispatch = useAppDispatch();
+  const navigate = useNavigate();
   const { users, dataStatus } = useAppSelector(({ users }) => ({
     users: users.users,
     dataStatus: users.dataStatus,
   }));
 
   const isRoot = pathname === AppRoute.ROOT;
 
+  const menuItems = useMemo(
+    () => [
```

Comment:
So far, I have added menu items to the App component as a mock. However, in the future, I plan to pass them from pages through PageLayout props